src/auth/components/form/link.tsx

```diff
@@ -0,0 +1,6 @@
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+export const StyledLink = styled(Link)`
+  margin: auto;
+` as typeof Link
```

Comment:
Przenieś to do `/form/container.tsx`

```suggestion
export const StyledLink = styled(Link)`
  margin: auto;
`
```